app/src/main/kotlin/model/io/neo4j/Neo4jRepositoryHandler.kt

```diff
@@ -0,0 +1,84 @@
+package model.io.neo4j
+
+import model.graphs.abstractGraph.Graph
+import org.neo4j.driver.AuthTokens
+import org.neo4j.driver.GraphDatabase
+
+object Neo4jRepositoryHandler {
+    var neo4jRepo: Neo4jRepository? = null
+        private set
+
+    var isRepoInit = false
+        private set
```

Comment:
💡 You don't need this as a separate property with its own backing field, you can just check `neo4jRepo != null`
```suggestion
    val isRepoInit: Boolean
        get() = neo4jRepo != null
```